tests/dialects/test_postgres.py

```diff
@@ -94,6 +94,7 @@ def test_postgres(self):
         self.validate_identity("COMMENT ON TABLE mytable IS 'this'")
         self.validate_identity("SELECT e'\\xDEADBEEF'")
         self.validate_identity("SELECT CAST(e'\\176' AS BYTEA)")
+        self.validate_identity("UPDATE MYTABLE AS T1 SET T1.COL = 13")
```

Comment:
This doesn't test the alias syntax you're trying to fix (`update table t1 ....`). You should instead use `validate_all`.